apps/rush-lib/src/api/PackageJsonEditor.ts

```diff
@@ -155,6 +155,10 @@ export class PackageJsonEditor {
     return new PackageJsonEditor(filename, object);
   }
 
+  public toObject(): IPackageJson {
```

Comment:
This is exposing a private data structure.  And because it is a shallow copy, it enables the caller to modify the data structure.

Reply:
Constructive suggestions:
- rename this to `saveToObject()` to clarify that nontrivial work is happening
- Call `_normalize()` to ensure the output is consistent with the editor state
- Deep clone the return value, so that modifications won't corrupt the `PackageJsonEditor`
- If possible, try to optimize the implementation in the common case where `PackageJsonEditor` is read but never modified (for example, maybe we can skip `_normalize()` entirely?)
- Write `/** */` docs that explain what this thing does